### ISSUE TITLE
Wrong description of Project Panel feature

### ISSUE THREAD

**Issue by dinkumoil** [notepad-plus-plus/npp-usermanual#135]
In chapter [Project Panels](https://github.com/notepad-plus-plus/npp-usermanual/blob/master/content/docs/session.md#project-panels) the Notepad++ manual states the following:

> For any entry in the Project Panel tree view, right clicking will give you the available actions on that specific entry.
> ...
> * On a Project entry: 
>   ...
>     * Add Files from Directory = add all the files from that Windows directory into the Project.
>       * **Please note: this is not a recursive action: this just puts all the files in the list.**

The statement above highlighted in bold is not correct. Indeed, selecting _Add Files from Directory_ works recursive and adds all files **and folders including their subfolders** to the project.

**Comment by pryrt** [comment]
@dinkumoil , I fixed that in #136.  Anything else I need to change before I merge that PR?

**Comment by dinkumoil** [comment]
Yes, I found some other minor "bugs", also in chapter [Project Panels](https://github.com/notepad-plus-plus/npp-usermanual/blob/master/content/docs/session.md#project-panels). 

**1. Obsolete member of unordered list left**

Since you deleted the first member of the unordered list this issue originally was about, the third member of this list is now obsolete:

> If you want a feature that automatically maps to the selected Windows directories, use Folder as Workspace instead.


**2. The second paragraph states:**

> Closing the tab for a file from the Project will not remove it from the Folder as Workspace panel, so it's easy to re-open that file.

I think that should be:

> Closing the tab for a file from the Project will not remove it from the **project panel**, so it's easy to re-open that file.


**3. The 6th paragraph states:**

> You can move a file into a Folder by dragging the file. Other than that, there aren't other drag-and-drop features in the Project Panel.

I was not able to drag & drop a file into a project panel, tested with Npp v7.9.1 32 bit. But maybe I did something wrong?


**4. Project files always contain relative file paths**
 
Project files always contain file paths relative to the location of the project file itself. Thus, if a project file is stored in a folder which is part of a project, it is very easy to move that folder to another location on the hard disk. If the project file is loaded from its new location it will still work. Maybe its worth to mention that somewhere.
